examples/hello_video/add_videos.py

```diff
@@ -0,0 +1,22 @@
+from aperturedb import Utils, ParallelLoader
+from dbinfo import DBInfo
+from WLASLDataKaggle import WSASLDataKaggle
+
+
+def main(helper: DBInfo):
+    dataset = WSASLDataKaggle(
+        records_count=helper.params.videos_count,
+    )
+    print(len(dataset))
+
+    loader = ParallelLoader.ParallelLoader(helper.create_connector())
+    loader.ingest(dataset, stats=True)
+
+
+if __name__ == "__main__":
+    main(DBInfo(mandatory_params={
```

Comment:
I think this is using old DBinfo stuff. Needs change. Maybe add this new example on a separete PR? the rest seems ready for merge.

Reply:
this is a file I havent finished working on . Got added by mistake. will remove from the PR